Jenkinsfile

```diff
@@ -0,0 +1,85 @@
+pipeline {
+  agent { label 'subman' }
+  stages {
+    // stage('prepare') {steps {echo 'prepare'}}
+    stage('Test') {
+      parallel {
+        stage('stylish') {
+          agent { label 'subman-centos7' }
+          steps { sh readFile(file: 'jenkins/stylish-tests.sh') }
+        }
+        stage('tito') {
+          agent { label 'rpmbuild' }
+          steps { sh readFile(file: 'jenkins/tito-tests.sh') }
+        }
+        stage('RHEL7 unit') {
+          agent { label 'subman-centos7' }
+          steps {
+            // echo "skipping for debug..."
+            sh readFile(file: 'jenkins/nose-tests.sh')
```

Comment:
The jenkins `sh` command creates a shell on a jenkins node,  running `sh './jenkins/nose-tests.sh'` would create a subshell in that shell which can get confusing for debugging certain things.  The `readFile()` function directly places the contents of that script into the shell.  There are benefits to subshells, but we're not making use of them, so I decided to avoid the ambiguity.  If we wanted to change to subshells we would need to add `#!/bin/bash` to the top of each file and run a `chmod +x` on the scripts.

Reply:
Thanks for info. Very useful! :+1: